src/main/java/org/mskcc/oncokb/curation/security/SecurityUtils.java

```diff
@@ -44,6 +47,11 @@ private static String extractPrincipal(Authentication authentication) {
             if (attributes.containsKey("preferred_username")) {
                 return (String) attributes.get("preferred_username");
             }
+        } else if (authentication.getPrincipal() instanceof DefaultOAuth2User) {
```

Comment:
@calvinlu3 what changed recently that we need to add this clause? I handle DefaultOidcUser